src/one_dragon/envs/ghproxy_service.py

```
@@ -15,6 +15,12 @@ def update_proxy_url(self) -> None:
         更新免费代理的url
         :return:
         """
+        # 如果 proxy_type 不是 'ghproxy'，则直接返回，不执行后续操作
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)